g2p_seq2seq/g2p.py

```diff
@@ -414,7 +478,10 @@ def decode(self, decode_lines, output_file=None):
     if output_file:
       for word in decode_lines:
         word = word.strip()
-        phonemes = self.decode_word(word)
+        if self.mode and self.mode == 'g2p':
```

Comment:
Like I wrote, it is not C, you can just use

```
if self.mode == 'g2p':
```

and that will be enough. self.mode will never be NULL like in C.

Reply:
Done!